python/src/scipp/plotting/figure.py

```diff
@@ -76,22 +77,22 @@ def _ipython_display_(self):
         """
         return self._to_widget()._ipython_display_()
 
-    def _to_widget(self, as_static=False):
+    def _to_widget(self):
         """
         Convert the Matplotlib figure to a widget. If the ipympl (widget)
         backend is in use, return the custom toolbar and the figure canvas.
         If not, convert the plot to a png image and place inside an ipywidgets
         Image container.
         """
         if self.is_widget():
-            if as_static:
-                return ipw.HBox([self.toolbar._to_widget(), self._to_image()])
+            if self.image is not None:
+                return ipw.HBox([self.toolbar._to_widget(), self.image])
             else:
                 return ipw.HBox([self.toolbar._to_widget(), self.fig.canvas])
         else:
             return self._to_image()
 
-    def _to_image(self):
+    def close(self):
```

Comment:
Not sure this function should be called close. Maybe `make_static` or `to_static` would be better?

Reply:
`persists()` maybe?

Reply:
i left it as `to_image`, it's more explicit. It's basically making it into an image